README.md
# Jekyllized Story by [HTML5 UP](https://html5up.net/story), [Stevo](https://github.com/smledbetter), and [Creative Directions](https://creativedirections.work)
This template is free for personal and commercial use under the [CCA 3.0 license](LICENSE.md)

**Jekyllized Story** is a Franken-theme created mostly from HTML5 UP's [Story](https://html5up.net/story) with a drawer menu and blog layout from [Phantom](https://html5up.net/phantom), and a custom header menu. Much of the Jekyllization is based on [andrewbanchich's Jekyllized Phantom Theme](https://github.com/andrewbanchich/phantom-jekyll-theme).

# Highlights:
- Extremely modular and tweakable, and as such is entirely
built on reusable, insanely customizable elements (like "spotlight" and "gallery") that
can be endlessly duplicated and tweaked into whatever you need to tell your story --
without, in many cases, touching *any* CSS.
- An interactive demo homepage.


# Examples in the wild
- [habitry.com](https://habitry.com)
- [smledbetter.com](https://smledbetter.com)

# Technical Notes
## Production
	Delete index-demo.html and assets/js/demo.js when you're done messing with the demo
	as neither are needed in production :)
## Feature list
* RSS feeds
* sitemap
* Multi-language support
* [text expand](https://jekyllcodex.org/without-plugin/text-expand/)

# Credits:

	Demo Images:
		Pexels](https://www.pexels.com/)

	Icons:
		[FontAwesome](https://fontawesome.com/)

	Other:
		jQuery (jquery.com)
		Misc. Sass functions (@HugoGiraudel)
		Skel (skel.io)

### Addition by [Stevo](https://github.com/smledbetter)
- Docker-Compose development environment.
- Custom headers.
- Custom gradient typography, tied to theme colors.
- Image overlay, "jekyll-story," tied to theme colors.
- Gradient icons, tied to theme colors.
- Additional spotlight layout.
- More transitions.
- More modifiers.
- Multiple pages and layouts.
- Multi-author blogging.
- Automatically generated Open Graph, Twitter, Schema.org, JSON-LD, SEO meta tags.
- Automatically generated Social Media footer.
- Updated to [FontAwesome 5](https://fontawesome.com/) via CDN.
- [Swiper slideshow](http://idangero.us/swiper/).
- [CSSgram](https://una.im/CSSgram/) custom image overlays.


pages/news.md
---
layout: page
title: Company News
nav_fname: nav_news


lang: en
lang-ref: news.md
---

<section class="wrapper style1 align-center invert">
  <div class="inner">
    <h2> Company News</h2>
    <!-- <p class="major">Here are some awesome blog posts.</p> -->
  </div>
</section>

{% include news_post.html %}



assets/js/main.js
/*
	Story by HTML5 UP
	html5up.net | @ajlkn
	Free for personal and commercial use under the CCA 3.0 license (html5up.net/license)
*/

(function($) {

	skel.breakpoints({
		xlarge: '(max-width: 1680px)',
		large: '(max-width: 1280px)',
		medium: '(max-width: 980px)',
		small: '(max-width: 736px)',
		xsmall: '(max-width: 480px)',
		xxsmall: '(max-width: 360px)'
	});

	$(function() {

		var	$window = $(window),
			$body = $('body'),
			$wrapper = $('#wrapper');

		// Disable animations/transitions until the page has loaded.
			$body.addClass('is-loading');

			$window.on('load', function() {
				window.setTimeout(function() {
					$body.removeClass('is-loading');
				}, 100);
			});

		// Fix: Placeholder polyfill.
			$('form').placeholder();

		// Prioritize "important" elements on medium.
			skel.on('+medium -medium', function() {
				$.prioritize(
					'.important\\28 medium\\29',
					skel.breakpoint('medium').active
				);
			});

		// Browser fixes.

			// IE: Flexbox min-height bug.
				if (skel.vars.browser == 'ie')
					(function() {

						var flexboxFixTimeoutId;

						$window.on('resize.flexbox-fix', function() {

							var $x = $('.fullscreen');

							clearTimeout(flexboxFixTimeoutId);

							flexboxFixTimeoutId = setTimeout(function() {

								if ($x.prop('scrollHeight') > $window.height())
									$x.css('height', 'auto');
								else
									$x.css('height', '100vh');

							}, 250);

						}).triggerHandler('resize.flexbox-fix');

					})();

			// Object fit workaround.
				if (!skel.canUse('object-fit'))
					(function() {

						$('.banner .image, .spotlight .image').each(function() {

							var $this = $(this),
								$img = $this.children('img'),
								positionClass = $this.parent().attr('class').match(/image-position-([a-z]+)/);

							// Set image.
								$this
									.css('background-image', 'url("' + $img.attr('src') + '")')
									.css('background-repeat', 'no-repeat')
									.css('background-size', 'cover');

							// Set position.
								switch (positionClass.length > 1 ? positionClass[1] : '') {

									case 'left':
										$this.css('background-position', 'left');
										break;

									case 'right':
										$this.css('background-position', 'right');
										break;

									default:
									case 'center':
										$this.css('background-position', 'center');
										break;

								}

							// Hide original.
								$img.css('opacity', '0');

						});

					})();

		// Smooth scroll.
			$('.smooth-scroll').scrolly();
			$('.smooth-scroll-middle').scrolly({ anchor: 'middle' });

		// Wrapper.
			$wrapper.children()
				.scrollex({
					top:		'30vh',
					bottom:		'30vh',
					initialize:	function() {
						$(this).addClass('is-inactive');
					},
					terminate:	function() {
						$(this).removeClass('is-inactive');
					},
					enter:		function() {
						$(this).removeClass('is-inactive');
					},
					leave:		function() {

						var $this = $(this);

						if ($this.hasClass('onscroll-bidirectional'))
							$this.addClass('is-inactive');

					}
				});

		// Items.
			$('.items')
				.scrollex({
					top:		'30vh',
					bottom:		'30vh',
					delay:		50,
					initialize:	function() {
						$(this).addClass('is-inactive');
					},
					terminate:	function() {
						$(this).removeClass('is-inactive');
					},
					enter:		function() {
						$(this).removeClass('is-inactive');
					},
					leave:		function() {

						var $this = $(this);

						if ($this.hasClass('onscroll-bidirectional'))
							$this.addClass('is-inactive');

					}
				})
				.children()
					.wrapInner('<div class="inner"></div>');

		// Gallery.
			$('.gallery')
				.wrapInner('<div class="inner"></div>')
				.prepend(skel.vars.mobile ? '' : '<div class="forward"></div><div class="backward"></div>')
				.scrollex({
					top:		'30vh',
					bottom:		'30vh',
					delay:		50,
					initialize:	function() {
						$(this).addClass('is-inactive');
					},
					terminate:	function() {
						$(this).removeClass('is-inactive');
					},
					enter:		function() {
						$(this).removeClass('is-inactive');
					},
					leave:		function() {

						var $this = $(this);

						if ($this.hasClass('onscroll-bidirectional'))
							$this.addClass('is-inactive');

					}
				})
				.children('.inner')
					//.css('overflow', 'hidden')
					.css('overflow-y', skel.vars.mobile ? 'visible' : 'hidden')
					.css('overflow-x', skel.vars.mobile ? 'scroll' : 'hidden')
					.scrollLeft(0);

			// Style #1.
				// ...

			// Style #2.
				$('.gallery')
					.on('wheel', '.inner', function(event) {

						var	$this = $(this),
							delta = (event.originalEvent.deltaX * 10);

						// Cap delta.
							if (delta > 0)
								delta = Math.min(25, delta);
							else if (delta < 0)
								delta = Math.max(-25, delta);

						// Scroll.
							$this.scrollLeft( $this.scrollLeft() + delta );

					})
					.on('mouseenter', '.forward, .backward', function(event) {

						var $this = $(this),
							$inner = $this.siblings('.inner'),
							direction = ($this.hasClass('forward') ? 1 : -1);

						// Clear move interval.
							clearInterval(this._gallery_moveIntervalId);

						// Start interval.
							this._gallery_moveIntervalId = setInterval(function() {
								$inner.scrollLeft( $inner.scrollLeft() + (5 * direction) );
							}, 10);

					})
					.on('mouseleave', '.forward, .backward', function(event) {

						// Clear move interval.
							clearInterval(this._gallery_moveIntervalId);

					});

			// Lightbox.
				$('.gallery.lightbox')
					.on('click', 'a', function(event) {

						var $a = $(this),
							$gallery = $a.parents('.gallery'),
							$modal = $gallery.children('.modal'),
							$modalImg = $modal.find('img'),
							href = $a.attr('href');

						// Not an image? Bail.
							if (!href.match(/\.(jpg|gif|png|mp4)$/))
								return;

						// Prevent default.
							event.preventDefault();
							event.stopPropagation();

						// Locked? Bail.
							if ($modal[0]._locked)
								return;

						// Lock.
							$modal[0]._locked = true;

						// Set src.
							$modalImg.attr('src', href);

						// Set visible.
							$modal.addClass('visible');

						// Focus.
							$modal.focus();

						// Delay.
							setTimeout(function() {

								// Unlock.
									$modal[0]._locked = false;

							}, 600);

					})
					.on('click', '.modal', function(event) {

						var $modal = $(this),
							$modalImg = $modal.find('img');

						// Locked? Bail.
							if ($modal[0]._locked)
								return;

						// Already hidden? Bail.
							if (!$modal.hasClass('visible'))
								return;

						// Lock.
							$modal[0]._locked = true;

						// Clear visible, loaded.
							$modal
								.removeClass('loaded')

						// Delay.
							setTimeout(function() {

								$modal
									.removeClass('visible')

								setTimeout(function() {

									// Clear src.
										$modalImg.attr('src', '');

									// Unlock.
										$modal[0]._locked = false;

									// Focus.
										$body.focus();

								}, 475);

							}, 125);

					})
					.on('keypress', '.modal', function(event) {

						var $modal = $(this);

						// Escape? Hide modal.
							if (event.keyCode == 27)
								$modal.trigger('click');

					})
					.prepend('<div class="modal" tabIndex="-1"><div class="inner"><img src="" /></div></div>')
						.find('img')
							.on('load', function(event) {

								var $modalImg = $(this),
									$modal = $modalImg.parents('.modal');

								setTimeout(function() {

									// No longer visible? Bail.
										if (!$modal.hasClass('visible'))
											return;

									// Set loaded.
										$modal.addClass('loaded');

								}, 275);

							});

							// Menu.
	var $menu = $('#menu');

	$menu.wrapInner('<div class="inner"></div>');

	$menu._locked = false;

	$menu._lock = function() {

		if ($menu._locked)
			return false;

		$menu._locked = true;

		window.setTimeout(function() {
			$menu._locked = false;
		}, 350);

		return true;

	};

	$menu._show = function() {

		if ($menu._lock())
			$body.addClass('is-menu-visible');

	};

	$menu._hide = function() {

		if ($menu._lock())
			$body.removeClass('is-menu-visible');

	};

	$menu._toggle = function() {

		if ($menu._lock())
			$body.toggleClass('is-menu-visible');

	};

	$menu
		.appendTo($body)
		.on('click', function(event) {
			event.stopPropagation();
		})
		.on('click', 'a', function(event) {

			var href = $(this).attr('href');

			event.preventDefault();
			event.stopPropagation();

			// Hide.
				$menu._hide();

			// Redirect.
				if (href == '#menu')
					return;

				window.setTimeout(function() {
					window.location.href = href;
				}, 350);

		})
		.append('<a class="close" href="#menu">Close</a>');

	$body
		.on('click', 'a[href="#menu"]', function(event) {

			event.stopPropagation();
			event.preventDefault();

			// Toggle.
				$menu._toggle();

		})
		.on('click', function(event) {

			// Hide.
				$menu._hide();

		})
		.on('keydown', function(event) {

			// Hide on escape.
				if (event.keyCode == 27)
					$menu._hide();

		});

	});

})(jQuery);


assets/js/demo.js
/*
	Story by HTML5 UP
	html5up.net | @ajlkn
	Free for personal and commercial use under the CCA 3.0 license (html5up.net/license)

	Note: Only needed for demo purposes. Delete for production sites.
*/

(function($) {

	var $window = $(window);

	// Styles.
		$(
			'<style>' +
				'.demo-animate-all:not(.gallery), .demo-animate-all:not(.gallery) *, .demo-animate-all:not(.gallery) *:before, .demo-animate-all:not(.gallery) *:after { transition: all 0.5s ease-in-out; }' +
				'.demo-controls .property .classes { display: none; }' +
				'.demo-controls .property[data-requires] { display: none; }' +
				'.demo-controls .property[data-requires].active { display: inline; }' +
				'.demo-controls .property .tooltip { position: relative; }' +
				'.demo-controls .property .tooltip:before { content: \'Click to change!\'; font-size: 0.7rem; position: absolute; bottom: 100%; left: 0; background: #47D3E5; color: #ffffff; line-height: 1; white-space: nowrap; font-weight: bold; border-radius: 0.125rem; padding: 0.325rem 0.425rem; animation: demo-controls-tooltip 1.5s forwards; animation-delay: 1s; opacity: 0; }' +
				'.demo-controls .property .tooltip:after { content: \'\'; position: absolute; bottom: calc(100% - 0.25rem); left: 0.5rem; border-left: solid 0.5rem transparent; border-right: solid 0.5rem transparent; border-top: solid 0.5rem #47D3E5; width: 0.5rem; height: 0.5rem; animation: demo-controls-tooltip 1.5s forwards; animation-delay: 1s; opacity: 0; }' +
				'@keyframes demo-controls-tooltip {' +
					'0% { opacity: 0; transform: translateY(0); }' +
					'10% { opacity: 1; transform: translateY(0.125rem); }' +
					'20% { opacity: 1; transform: translateY(-0.125rem); }' +
					'30% { opacity: 1; transform: translateY(0.125rem); }' +
					'40% { opacity: 1; transform: translateY(-0.125rem); }' +
					'50% { opacity: 1; transform: translateY(0.125rem); }' +
					'60% { opacity: 1; transform: translateY(0); }' +
					'90% { opacity: 1; }' +
					'100% { opacity: 0; }' +
				'}' +
			'</style>'
		).appendTo($('head'));

	// Functions.
		$.fn.demo_controls = function(styles, userOptions) {

			var $this = $(this),
				$styleProperty, $stylePropertyClasses,
				$controls, $x, $y, $z,
				options,
				current, i, j, k, s, n, count;

			// No elements?
				if (this.length == 0)
					return $this;

			// Multiple elements?
				if (this.length > 1) {

					for (var i=0; i < this.length; i++)
						$(this[i]).demo_controls(styles, userOptions);

					return $this;

				}

			// Options.
				options = $.extend({
					target: null,
					palette: true
				}, userOptions);

			// Controls.
				if (styles) {

					$controls = $(
						'<span class="demo-controls">' +
							'<span class="property" data-name="style">' +
								'<a href="#" class="title tooltip">style</a>' +
								'<span class="classes"></span>' + (options.palette ? ', ' : ' ') +
							'</span>' +
							(options.palette ?
								'<span class="property active" data-name="scheme">' +
									'<a href="#" class="title">scheme</a>' +
									'<span class="classes">' +
										'<span data-class="-" class="active">default</span>' +
										'<span data-class="invert">invert</span>' +
									'</span>, ' +
								'</span>' +
								'<span class="property active" data-name="color">' +
									'<a href="#" class="title">color</a>' +
									'<span class="classes">' +
										'<span data-class="-" class="active">default</span>' +
										'<span data-class="color1">color1</span>' +
										'<span data-class="color2">color2</span>' +
										'<span data-class="color3">color3</span>' +
										'<span data-class="color4">color4</span>' +
										'<span data-class="color5">color5</span>' +
										'<span data-class="color6">color6</span>' +
										'<span data-class="color7">color7</span>' +
									'</span>, ' +
								'</span>'
							: '') +
						'</span>'
					);

				}
				else {

					$controls = $(
						'<span class="demo-controls">' +
							(options.palette ?
								'<span class="property active" data-name="scheme">' +
									'<a href="#" class="title">scheme</a>' +
									'<span class="classes">' +
										'<span data-class="-" class="active">default</span>' +
										'<span data-class="invert">invert</span>' +
									'</span> and ' +
								'</span>' +
								'<span class="property active" data-name="color">' +
									'<a href="#" class="title">color</a>' +
									'<span class="classes">' +
										'<span data-class="-" class="active">default</span>' +
										'<span data-class="color1">color1</span>' +
										'<span data-class="color2">color2</span>' +
										'<span data-class="color3">color3</span>' +
										'<span data-class="color4">color4</span>' +
										'<span data-class="color5">color5</span>' +
										'<span data-class="color6">color6</span>' +
										'<span data-class="color7">color7</span>' +
									'</span>' +
								'</span>'
							: '') +
						'</span>'
					);

				}

			// Target.
				switch (options.target) {

					case 'previous':
						$this.prev().find('.demo-controls').replaceWith($controls);
						break;

					default:
						$this.find('.demo-controls').replaceWith($controls);
						break;

				}

			// Styles.
				if (styles) {

					$styleProperty = $controls.find('.property[data-name="style"]');
					$stylePropertyClasses = $styleProperty.children('.classes');

					for (i in styles) {

						current = false;
						count = Object.keys(styles[i]).length;
						n = 1;

						// Add to style property.
							$x = $('<span data-class="' + i + '">, ' + i + '</span>')
								.appendTo($stylePropertyClasses);

							if ($this.hasClass(i)) {

								$x.addClass('active');
								current = true;

							}

						// Step through properties.
							for (j in styles[i]) {

								$x = $(
									'<span class="property" data-name="' + j + '" data-requires="' + i + '">' +
										(n == count ? '<span>and </span>' : '') +
										'<a href="#" class="title">' + j + '</a>' +
										'<span class="classes">' +
										'</span>' + (n < count ? ', ' : '') +
									'</span>'
								).appendTo($controls);

								$y = $x.children('.classes');

								if (current)
									$x.addClass('active');

								for (k in styles[i][j]) {

									$z = $('<span data-class="' + k + '">, ' + styles[i][j][k].replace('*', '') + '</span>')
										.appendTo($y);

									if (styles[i][j][k].substr(-1, 1) == '*')
										$z.addClass('default');

									if (current
									&&	$this.hasClass(k))
										$z.addClass('active');

								}

								n++;

							}

					}

				}

			// Events.
				$controls.on('click', 'a', function(event) {
					event.preventDefault();
				});

				$controls.on('click', '.property.active', function(event) {

					var $property = $(this);
					var $classes = $property.find('.classes > *');
					var $current = $classes.filter('.active');
					var $next;

					// Determine next.
						if ($current.length == 0
						||	$current.index() == $classes.length - 1)
							$next = $classes.first();
						else
							$next = $current.next();

					// Turn on animate all.
						$this.addClass('demo-animate-all');

					// Deactivate current.
						$current.removeClass('active');
						$this.removeClass($current.data('class'));

					// Activate next.
						$next.addClass('active');
						$this.addClass($next.data('class'));

					// Turn off animate all.
						setTimeout(function() {
							$this.removeClass('demo-animate-all');
						}, 500);

				});

				$controls.on('click', '.property[data-name="style"]', function(event) {

					var $property = $(this);
					var $classes = $property.find('.classes > *');
					var $current = $classes.filter('.active');
					var $next;

					// Determine next.
						if ($current.length == 0
						||	$current.index() == $classes.length - 1)
							$next = $classes.first();
						else
							$next = $current.next();

					// Turn on animate all.
						$this.addClass('demo-animate-all');

					// Deactivate current.
						$current.removeClass('active');
						$this.removeClass($current.data('class'));

						$controls.find('.property[data-requires="' + $current.data('class') + '"]')
							.removeClass('active');

						$controls.find('.property[data-requires="' + $current.data('class') + '"] > .classes > .active').each(function() {

							$(this).removeClass('active');

							if ($(this).data('class') != '-')
								$this.removeClass($(this).data('class'));

						});

					// Activate next.
						$next.addClass('active');
						$this.addClass($next.data('class'));

						$controls.find('.property[data-requires="' + $next.data('class') + '"]')
							.addClass('active');

						$controls.find('.property[data-requires="' + $next.data('class') + '"] > .classes > .default').each(function() {

							$(this).addClass('active');

							if ($(this).data('class') != '-')
								$this.addClass($(this).data('class'));

						});

					// Turn off animate all.
						setTimeout(function() {
							$this.removeClass('demo-animate-all');
						}, 500);

				});

		};

	// Elements.

		// Wrappers.
			$('.wrapper').demo_controls({
				style1: {
					'content alignment': {
						'align-left': 'left*',
						'align-center': 'center',
						'align-right': 'right'
					}
				},
				style2: {
					'content alignment': {
						'align-left': 'left',
						'align-center': 'center*',
						'align-right': 'right'
					},
				},
			});

		// Banner.
			$('.banner').demo_controls({
				style1: {
					'size': {
						'-': 'normal',
						'fullscreen': 'fullscreen*'
					},
					'orientation': {
						'orient-left': 'left*',
						'orient-right': 'right'
					},
					'content alignment': {
						'content-align-left': 'left*',
						'content-align-center': 'center',
						'content-align-right': 'right'
					},
					'image position': {
						'image-position-left': 'left',
						'image-position-center': 'center*',
						'image-position-right': 'right'
					}
				},
				style2: {
					'size': {
						'-': 'normal',
						'fullscreen': 'fullscreen*'
					},
					'orientation': {
						'orient-left': 'left',
						'orient-center': 'center*',
						'orient-right': 'right'
					},
					'content alignment': {
						'content-align-left': 'left',
						'content-align-center': 'center*',
						'content-align-right': 'right'
					},
					'image position': {
						'image-position-left': 'left',
						'image-position-center': 'center*',
						'image-position-right': 'right'
					}
				},
				style3: {
					'size': {
						'-': 'normal',
						'fullscreen': 'fullscreen*'
					},
					'orientation': {
						'orient-left': 'left',
						'orient-right': 'right*'
					},
					'content alignment': {
						'content-align-left': 'left*',
						'content-align-center': 'center',
						'content-align-right': 'right'
					},
					'image position': {
						'image-position-left': 'left',
						'image-position-center': 'center*',
						'image-position-right': 'right'
					}
				},
				style4: {
					'size': {
						'-': 'normal',
						'fullscreen': 'fullscreen*'
					},
					'phone type': {
						'iphone': 'iphone*',
						'android': 'android'
					},
					'orientation': {
						'orient-left': 'left',
						'orient-right': 'right*'
					},
					'content alignment': {
						'content-align-left': 'left*',
						'content-align-center': 'center',
						'content-align-right': 'right'
					},
					'image position': {
						'image-position-left': 'left',
						'image-position-center': 'center*',
						'image-position-right': 'right'
					}
				},
				style5: {
					'size': {
						'-': 'normal',
						'fullscreen': 'fullscreen*'
					},
					'content alignment': {
						'content-align-left': 'left',
						'content-align-center': 'center*',
						'content-align-right': 'right'
					},
					'image position': {
						'image-position-left': 'left',
						'image-position-center': 'center*',
						'image-position-right': 'right'
					}
				}
			});

		// Spotlight.
			$('.spotlight').demo_controls({
				style1: {
					'orientation': {
						'orient-left': 'left',
						'orient-right': 'right*'
					},
					'content alignment': {
						'content-align-left': 'left*',
						'content-align-center': 'center',
						'content-align-right': 'right'
					},
					'image position': {
						'image-position-left': 'left*',
						'image-position-center': 'center',
						'image-position-right': 'right'
					}
				},
				style2: {
					'orientation': {
						'orient-left': 'left',
						'orient-right': 'right*'
					},
					'content alignment': {
						'content-align-left': 'left*',
						'content-align-center': 'center',
						'content-align-right': 'right'
					},
					'image position': {
						'image-position-left': 'left',
						'image-position-center': 'center*',
						'image-position-right': 'right'
					}
				},
				style3: {
					'phone type': {
						'iphone': 'iphone*',
						'android': 'android'
					},
					'orientation': {
						'orient-left': 'left',
						'orient-right': 'right*'
					},
					'content alignment': {
						'content-align-left': 'left*',
						'content-align-center': 'center',
						'content-align-right': 'right'
					},
					'image position': {
						'image-position-left': 'left',
						'image-position-center': 'center*',
						'image-position-right': 'right'
					}
				},
				style4: {
					'size': {
						'-size': 'normal',
						'fullscreen': 'fullscreen*',
						'halfscreen': 'halfscreen'
					},
					'orientation': {
						'orient-left': 'left*',
						'orient-center': 'center',
						'orient-right': 'right'
					},
					'content alignment': {
						'content-align-left': 'left*',
						'content-align-center': 'center',
						'content-align-right': 'right'
					},
					'image position': {
						'image-position-left': 'left',
						'image-position-center': 'center*',
						'image-position-right': 'right'
					}
				},
				style5: {
					'size': {
						'-size': 'normal',
						'fullscreen': 'fullscreen*',
						'halfscreen': 'halfscreen'
					},
					'orientation': {
						'orient-left': 'left*',
						'orient-center': 'center',
						'orient-right': 'right'
					},
					'content alignment': {
						'content-align-left': 'left*',
						'content-align-center': 'center',
						'content-align-right': 'right'
					},
					'image position': {
						'image-position-left': 'left',
						'image-position-center': 'center*',
						'image-position-right': 'right'
					}
				},
				style6: {
					'size': {
						'-size': 'normal',
						'fullscreen': 'fullscreen*',
						'halfscreen': 'halfscreen'
					},
					'orientation': {
						'orient-left': 'left*',
						'orient-center': 'center',
						'orient-right': 'right'
					},
					'content alignment': {
						'content-align-left': 'left*',
						'content-align-center': 'center',
						'content-align-right': 'right'
					},
					'image position': {
						'image-position-left': 'left',
						'image-position-center': 'center*',
						'image-position-right': 'right'
					}
				},
			});

		// Gallery.
			$('.gallery').demo_controls({
				style1: {
					'size': {
						'xsmall':'xsmall',
						'small': 'small',
						'medium': 'medium*',
						'big': 'big'
					}
				},
				style2: {
					'size': {
						'xsmall':'xsmall',
						'small': 'small',
						'medium': 'medium*',
						'big': 'big'
					}
				},
			}, {
				target: 'previous',
				palette: false
			});

		// Items.
			$('.items').demo_controls({
				style1: {
					'size': {
						'small': 'small',
						'medium': 'medium*',
						'big': 'big'
					}
				},
				style2: {
					'size': {
						'small': 'small',
						'medium': 'medium*',
						'big': 'big'
					}
				},
				style3: {
					'size': {
						'small': 'small',
						'medium': 'medium*',
						'big': 'big'
					}
				}
			}, {
				target: 'previous',
				palette: false
			});

})(jQuery);
